### PR TITLE
Add free courses in pt-br for Java

### PR DESCRIPTION
**Pull request by gabirlima**
Adding 5 free courses for Java in Portuguese-BR from Coursera, developed by a highly recognized university in Brazil, ITA.
The courses will add a lot of knowledge for beginners as well as advanced developers. 
On the main screen there is already a button saying that it is free. If necessary, you can take a test and obtain a certificate paying a symbolic fee.

### Checklist:
- [x ] Not a duplicate
- [x ] Included author(s) if appropriate
- [ x] Lists are in alphabetical order
- [ x] Needed indications added (PDF, access notes, under construction)

### CODE REVIEW

File: free-courses-pt_BR.md [hunk 1]
```diff
@@ -121,13 +121,17 @@
...
 * [Introdução ao Java e Orientação a objetos](https://www.udemy.com/introducao-ao-java-e-orientacao-a-objetos/) - Helder Guimaraes Aragao (Udemy)
 * [Java SE - Polimorfismo](https://www.udemy.com/java-se-polimorfismo/) - Fernando Franzini (Udemy)
+* [Orientação a Objetos com Java](https://www.coursera.org/learn/orientacao-a-objetos-com-java) - Eduardo Guerra e Clovis Fernandes - ITA (Coursera)
+* [Princípios de Desenvolvimento Ágil de Software](https://www.coursera.org/learn/principios-de-desenvolvimento-agil-de-software) - Eduardo Guerra e Clovis Fernandes - ITA (Coursera)
 * [Produtos Java - Especificações versus Proprietários](https://www.udemy.com/produtos-java-especificacoes-versus-proprietarios/) - Fernando Franzini (Udemy)
+* [TDD – Desenvolvimento de software guiado por testes](https://www.coursera.org/learn/tdd-desenvolvimento-de-software-guiado-por-testes) - Eduardo Guerra e Clovis Fernandes - ITA (Coursera)
 * [Testes unitários com Java utilizando o Junit](https://www.udemy.com/testes-unidade-automaticos-software-junit/) - Gustavo Farias (Udemy)
 
-
```
Comment (SethiPandi) [inline free-courses-pt_BR.md:130]: There should be two line breaks between each section.

File: .idea/.gitignore [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```
Comment (SethiPandi) [inline .idea/.gitignore:1]: I think you have committed your IDEA files by accident, you may want to remove them.

**Review by SethiPandi — COMMENTED** [review]
You can read the contribution guide to get any information on how this repository should be formatted!

File: .idea/.gitignore (hunk 2 above)
Comment (gabirlima) [reply .idea/.gitignore]: OK

File: free-courses-pt_BR.md (hunk 1 above)
Comment (gabirlima) [reply free-courses-pt_BR.md:130]: OK

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**